genesis_zktrie.json

```diff
@@ -0,0 +1,73 @@
+{
+  "config": {
+    "chainId": 53077,
+    "homesteadBlock": 0,
+    "eip150Block": 0,
+    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+    "eip155Block": 0,
+    "eip158Block": 0,
+    "byzantiumBlock": 0,
+    "constantinopleBlock": 0,
+    "petersburgBlock": 0,
+    "istanbulBlock": 0,
+    "berlinBlock": 0,
+    "londonBlock": 0,
+    "clique": {
+      "period": 15,
+      "epoch": 30000
+    },
+    "zktrie": true
```

Comment:
Is it possible to use the same genesis.json file, and choose zktrie module by cmd params  i.g: 'geth --zktrie' ?

Reply:
Cmd params may be not feasible because the data being persisted in db is complete different for zktrie. Run geth with zktrie on data saved without this flag just ruin the whole db and vice visa. It has to be a flag in chain's config.

Reply:
i.g: `./build/bin/geth init --zktrie`, can add the flag at [here](https://github.com/scroll-tech/go-ethereum/blob/f73142728206ddc4b89d3b3e9b5549933eba94fe/cmd/geth/chaincmd.go#L51).

Reply:
I think it is ok only being applied in init sub command. But the genesis object in init is mandatory, as the result it may be a confliction if we have specified zktrie flag both in genesis object and cmd flag and we need a rule for respecting which one.

I found the original geth cmd do not handle such an issue by put all possible flags only avaliable in the genesis object. So should we really need such a flag for cmd (even for the possible risk of causing some confusion)?